doc/mgr/prometheus.rst

```diff
@@ -41,13 +43,21 @@ configuration parameter. The parameter is a comma or space separated list
 of ``pool[/namespace]`` entries. If the namespace is not specified the
 statistics are collected for all namespaces in the pool.
 
+Example to enable the images ``image1``, ``image2`` and ``imageN``::
+
+  ceph config set mgr mgr/prometheus/rbd_stats_pools "image1,image2,imageN"
+
```

Comment:
This is not a list of RBD images, but of RBD-enabled Pools.

Reply:
Okay will be fixed with the next push.